src/java/org/apache/cassandra/db/SSTableImporter.java

```diff
@@ -481,6 +502,8 @@ public String toString()
                    ", invalidateCaches=" + invalidateCaches +
                    ", extendedVerify=" + extendedVerify +
                    ", copyData= " + copyData +
+                   ", failOnMissingIndex= " + failOnMissingIndex +
+                   ", validateIndexChecksum= " + validateIndexChecksum +
```

Comment:
I'll come back and do a final review of all this stuff once we settle on naming ;)